sdcclient/_scanning.py

```diff
@@ -163,6 +163,52 @@ def query_image_vuln(self, image, vuln_type="", vendor_only=True):
         '''
         return self._query_image(image, query_group='vuln', query_type=vuln_type, vendor_only=vendor_only)
 
+    def query_images_by_vulnerability(self, vulnerability_id, namespace=None, package=None, severity=None, vendor_only=True):
+        '''**Description**
+            Search system for images with the given vulnerability ID present
+
+        **Arguments**
+            - vulnerability_id: Search for images vulnerable to this vulnerability ID (e.g. CVE-1999-0001)
+            - namespace: Filter results to images with vulnerable packages in the given namespace (e.g. debian:9)
+            - package: Filter results to images with the given vulnerable package name (e.g. sed)
+            - severity: Filter results to images with the given vulnerability severity (e.g. Medium)
+            - vendor_only: Only show images with vulnerabilities explicitly deemed applicable by upstream OS vendor, if present
+
+        **Success Return Value**
+            A JSON object representing the images.
+        '''
+        url = "{base_url}/api/scanning/v1/anchore/query/images/by_vulnerability?vulnerability_id={vulnerability_id}{namespace}{package}{severity}&vendor_only={vendor_only}".format(
+            base_url=self.url,
+            vulnerability_id=vulnerability_id,
+            namespace="&namespace={}".format(namespace) if namespace else "",
+            package="&affected_package={}".format(package) if package else "",
+            severity="&severity={}".format(severity) if severity else "",
+            vendor_only=vendor_only)
+
+        res = requests.get(url, headers=self.hdrs, verify=self.ssl_verify)
+        return self._request_result(res)
```

Comment:
Generally speaking, I'm not a big fan of how we manage the server responses across the board (I find it a little hard to use from outside), but for now that's the default behavior. Could you please consider following the same pattern we use elsewhere, eg. https://github.com/draios/python-sdc-client/blob/28f51f129e540d1e0cde7d2362384cea6ba70d5b/sdcclient/_scanning.py#L117-L120.

Thanks!